### ISSUE TITLE
user_token: ERROR_LOGIN_PARAMETERS_MISSING

### ISSUE THREAD

**Issue by humboldtux** [glpi-project/glpi#850]
I keep getting the same error when trying to access GLPI API via user token:

```
$ curl -X GET -H 'Content-Type: application/json' -H 'Authorization: user_token k4r2habhcruqrcz0ahiq285ci49t8ckrfctxm5ezv' 'http://glpi.lan:8080/api/initSession'
["ERROR_LOGIN_PARAMETERS_MISSING","le(s) paramètre(s) login, password ou user_token sont manquants; Afficher la documentation dans votre navigateur à http://glpi.lan:8080/api/#ERROR_LOGIN_PARAMETERS_MISSING"]
```

I don't have this error when using basic auth.

I'm using GLPI 9.1-RC2.

**Comment by Syldeweerdt** [comment]
Same issue

**Comment by orthagh** [comment]
Could you post your server configuration ?

I just tested with current master (not so far of RC2): 

```
$ curl -X GET -H 'Content-Type: application/json' -H 'Authorization: user_token ********' 'http://localhost/glpi/9.1/apirest.php/initSession'
{"session_token":"09idgvt20vd0gibhbm4teqmj80"}
```

**Comment by Slydew** [comment]
I must set App-token : 
curl -X GET -H "Content-Type: application/json" -H "App-Token: azertyuiop" -H "Authorization: user_token azertyuiop" http://myhost/glpi/api/initSession
["ERROR_LOGIN_PARAMETERS_MISSING","le(s) param├¿tre(s) login, password ou user_token sont manquants; Afficher la documentation dans votre navigateur ├á http://sos/api/#ERROR_LOGIN_PARAMETERS_MISSING"]

**Comment by Slydew** [comment]
I remove apptoken from setting, and now : ERROR_LOGIN_PARAMETERS_MISSING

What config file do you need on my xamp ?

**Comment by orthagh** [comment]
your environnement
- OS (with version)
- web server
- if your have specific tweaks

**Comment by orthagh** [comment]
Ok i found the error on xampp env.

I just push a fix (5bd4284) on master to avoid theses errors
